po/es.po

```diff
@@ -0,0 +1,706 @@
+msgid "can't open clone dir: {}"
+msgstr "no se puede abrir el directorio clonado: {}"
+
+#: src/command_line.rs:127
+msgid "option {} expects a value"
+msgstr "opción {} espera un valor"
+
+#: src/command_line.rs:158 src/command_line.rs:327
+msgid "option {} does not allow a value"
+msgstr "opción {} no admite un valor"
+
+#: src/command_line.rs:318
+msgid "unknown option --{}"
+msgstr "opción desconocida --{}"
+
+#: src/command_line.rs:321
+msgid "unknown option -{}"
+msgstr "opción desconocida -{}"
+
+#: src/completion.rs:27
+msgid "failed to open cache file '{}'"
+msgstr "error al abrir desde caché el fichero '{}'"
+
+#: src/completion.rs:65
+msgid "could not update aur cache"
+msgstr "no se puedo actualizar la caché de aur"
+
+#: src/config.rs:174
+msgid "invalid value '{:val}' for key '{:key}', expected: {:exp}"
+msgstr "valor inválido '{:val}' para clave '{:key}', se esperaba: {:exp}"
+
+#: src/config.rs:474
+msgid "failed to find cache directory"
+msgstr "error al encontrar el directorio de caché"
+
+#: src/config.rs:477
+msgid "failed to find config directory"
+msgstr "error al encontrar el directorio de configuración"
+
+#: src/config.rs:501
+msgid "config file '{}' does not exist"
+msgstr "no existe el fichero de configuración '{}'"
+
+#: src/config.rs:550
+msgid "only one operation may be used at a time"
+msgstr "solo se puede usar una operación a la vez"
+
+#: src/config.rs:638
+msgid "can not find local repo '{}' in pacman.conf"
+msgstr "no se puede encontrar el repositorio local '{}' en pacman.conf"
+
+#: src/config.rs:670
+msgid "failed to initialize alpm: root={} dbpath={}"
+msgstr "error al inicializar alpm: root={} dbpath={}"
+
+#: src/config.rs:740 src/config.rs:873
+msgid "value can not be empty for key '{}'"
+msgstr "la clave '{}' no admite un valor vacio"
+
+#: src/config.rs:756
+msgid "key '{}' does not belong to a section"
+msgstr "clave '{}' no pertenece a sección"
+
+#: src/config.rs:762
+msgid "unknown section '{}'"
+msgstr "sección desconocida '{}'"
+
+#: src/config.rs:769 src/config.rs:824 src/config.rs:829
+msgid "key can not be empty"
+msgstr "clave no puede estar vacia"
+
+#: src/config.rs:791
+msgid "error: unknown option '{}' in section [bin]"
+msgstr "error: opción desconocida '{}' es sección [bin]"
+
+#: src/config.rs:891
+msgid "error: unknown option '{}' in section [options]"
+msgstr "error: opción desconocida '{}' es sección [options]"
+
+#: src/config.rs:896
+msgid "option '{}' does not take a value"
+msgstr "opción {} no admite un valor"
+
+#: src/config.rs:947 src/install.rs:1796
+msgid "There are {:n} providers available for {:pkg}:"
+msgstr "Hay {:n} proveedores para {:pkg}"
+
+#: src/config.rs:961 src/info.rs:56 src/info.rs:95 src/install.rs:1755
+#: src/install.rs:1804
+msgid "Repository"
+msgstr "Repositorio"
+
+#: src/config.rs:983
+msgid "  failed to sync {}"
+msgstr " error al sincronizar {}"
+
+#: src/devel.rs:129 src/download.rs:179
+msgid "Querying AUR..."
+msgstr "Consultando AUR..."
```

Comment:
```suggestion
msgstr "Buscando el AUR..."
```

Reply:
"Buscando el AUR" is more like "Search the AUR". If using the word "Buscando" I would rather use it as "Buscando en el AUR"

Reply:
That's acceptable to me.